### DISCUSSION TITLE
How to use tailwindcss to generate dynamic class names with hash after project build

### DISCUSSION THREAD

**Discussion by arianshi** [tailwindlabs/tailwindcss#12438] [Help]
Hi tailwindcss team,

@josevalim  @wongjn

we want to use tailwindcss to generate dynamic class names with hash after build：

<img width="700" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/133647949/9bfed5bd-cf5b-4b95-846e-35d577c4f3ef">

<img width="717" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/133647949/f6468549-f862-48e3-899c-1e449cc515c9">

Currently, tailwindcss supports such CSS class names. Can this dynamic class name also be supported?

**Comment by xrkmed** [comment]
up
